SQC-352 SQC-353 create cert command documantion for mtls/CA cert chain management (#19205)
cloudflare/cloudflare-docs: src/content/docs/workers/wrangler/commands.mdx

---

## `cert`

> Manage mTLS client certificates and Certificate Authority (CA) chain certificates used for secured connnections.

Reviewer: ```suggestion
Manage mTLS client certificates and Certificate Authority (CA) chain certificates used for secured connections.
```

> These certificates can be used in Hyperdrive, which allow a Hyperdrive configuration to present the certificate when establishing a connection with an origin database that requires client authentication (mTLS) or custom Certificate Authority (CA).

Reviewer: ```suggestion
These certificates can be used in Hyperdrive configurations, enabling them to present the certificate when connecting to an origin database that requires client authentication (mTLS) or a custom Certificate Authority (CA).
```
